File: mifosng-android/src/main/java/com/mifos/objects/organisation/InterestCalculationDaysInYearType.java
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
package com.mifos.objects.organisation;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;

/**
 * Created by nkiboi on 12/15/2015.
 */
public class InterestCalculationDaysInYearType {
    private Integer id;
    private String code;
    private String value;


    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }


    @Override
    public String toString() {
        return "interestCalculationDaysInYearTypeOptions{" +
                "id=" + id +
                ", code='" + code + '\'' +
                ", value='" + value + '\'' +
                '}';
    }
}

File: core/datastore/src/main/java/com/mifos/core/objects/accounts/savings/InterestCalculationDaysInYearType.kt
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
package com.mifos.core.objects.accounts.savings

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
class InterestCalculationDaysInYearType(
    var id: Int? = null,

    var code: String? = null,

    var value: String? = null
) : Parcelable